demo/settings.py

```diff
@@ -6,6 +6,8 @@
     "demo.middleware.LiveDemoModeMiddleware",
 ]
 
+ROOT_URLCONF = "demo.urls"
```

Comment:
I'm glad you were able to avoid toggles in the main application. I think you might need to add a toggle for the `common_data` addition, though.